src/main/java/uk/gov/gchq/palisade/resource/AbstractLeafResource.java

```diff
@@ -37,48 +36,76 @@ public AbstractLeafResource() {
     }
 
     public AbstractLeafResource type(final String type) {
```

Comment:
This needs the @generated annotation on the mutator

Reply:
Done